src/doc/man/options-packages.adoc

```diff
@@ -1,9 +1,14 @@
 By default, when no package selection options are given, the packages selected
-depend on the current working directory. In the root of a virtual workspace,
-all workspace members are selected (`--all` is implied). Otherwise, only the
-package in the current directory will be selected. The default packages may be
-overridden with the `workspace.default-members` key in the root `Cargo.toml`
-manifest.
+depend on the selected manifest file (based on the current working directory if
+`--manifest-path` is not given). If the manifest is the root of a workspace then
+the workspaces default members are selected, otherwise only the package defined
+by the manifest will be selected.
+
+The default members of a workspace can either be set explicitly with the
```

Comment:
```suggestion
The default members of a workspace can be set explicitly with the
```

I think *either* used in this way needs two clauses within the same sentence.  It reads a little more clearly to me without it.